packages/modules/tax/src/migrations/Migration20241212052837.ts

```diff
@@ -0,0 +1,27 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20241212052837 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('alter table if exists "tax_provider" add column if not exists "created_at" timestamptz not null default now(), add column if not exists "updated_at" timestamptz not null default now(), add column if not exists "deleted_at" timestamptz null;');
```

Comment:
These two columns will be created now since they were not defined on the MikroORM model